File: src/main/java/com/sao/service/NewsService.java
package com.sao.service;

import com.sao.domain.model.News;

public interface NewsService {

    //get some news
    void saveNews();

    //fun news for banner with imageUrl
    Iterable<News> findBannerNews();

    //find plain news without imageUrl
    Iterable<News> findPlainNews();


    default boolean compare(Long a, Long b){
        if(a==null||b==null) return false;

        if(a.intValue() == b.intValue()){
            return true;
        }else{
            return false;
        }
    }
}

File: src/main/java/com/sao/service/NewsService.kt
package com.sao.service

import com.sao.domain.model.News

interface NewsService{


    //get some news
    fun saveNews()

    //fun news for banner with imageUrl
    fun findBannerNews():Iterable<News>

    //find all news
    fun findAllNews():Iterable<News>

    //find plain news without imageUrl
    fun findPlainNews():Iterable<News>

}